test/e2e/integration/Events/Events.feature

```diff
@@ -0,0 +1,26 @@
+Feature: Events
+
+  As a citizen I can view information for all available content on an event page
+  Prerequisite: Events Simple Test 1-BE-event-1
+
+  # Before hook
+  # - Logs into backend
+  # - Creates eventpage from fixture data @pageData and saves node id as @nodeId
+
+  # After hook
+  # - Deletes node id @nodeId
+
+  Scenario: Event Page loads correctly
+    When I have navigated to the created page
+    Then the page title should be "2-BE-event-1"
+    And the Description should be "Now Neptune had gone off to the Ethiopians, who are at the world's end, and lie in two halves, the one looking West and the other East. He had gone there to accept a hecatomb of sheep and oxen, and was enjoying himself at his festival; but the other gods met in the house of Olympian Jove, and the sire of gods and men spoke first."
+    And the Dates should be "20 May 2019 11:02 am - 13 May 2020 11:11 am"
+    And the Address should be "45 Collins St,  Melbourne, VIC 3000"
+    And the Price should be "34.33 - 89.95"
+    And the Web url should be "https://www.weatherzone.com.au/vic/central/healesville"
+    And the Requirements the should be "Accessible venue"
```

Comment:
And the Requirements ~the~ should be "Accessible venue"

Reply:
Fixed.